canvas_exercise/shapes_game/index.js

```diff
@@ -1,36 +1,156 @@
 window.addEventListener("load", function() {
 
-  function clear(ctx, width, heigt) {
+  var shapeIndex = [0, 1, 2, 3];
+  var shapes = ["orangeRec", "whiteRec", "orangeTri", "whiteTri"];
+
+  function getRandomIndex(arr) {
+      return arr[Math.floor(Math.random() * arr.length)]
+  }
+
+  function drawRandomShape(ctx, width, height, func){
+
+      index = getRandomIndex(shapeIndex); // gets a random number from our list 
```

Comment:
You could remove the shapeIndex array and just write  a function that returns a random number between 0 and 3.